microsoftgraph/microsoft-graph-explorer
# Feat/info banner

## Overview

Shows an informational message to users when querying
`/v1.0/me`
`/v1.o/users`

### Demo

![ge classic](https://user-images.githubusercontent.com/12011447/58876368-8e4c5000-86d6-11e9-85aa-921d5c4e7b9b.JPG)

### Notes
N/A

## Testing Instructions

* Launch Graph Explorer
* Make a `GET` request to `/v1.0/me`
* Observe that an info message is shown

Fixes [284](https://github.com/microsoftgraph/microsoft-graph-explorer/issues/284)
